crates/control/migrations/20220125162950_create_connector_images.up.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE connector_images (
+  "id" BIGINT NOT NULL DEFAULT id_generator(),
+  PRIMARY KEY (id),
+  "connector_id" BIGINT NOT NULL REFERENCES connectors("id"),
+  "image" TEXT NOT NULL,
+  "tag" TEXT NOT NULL,
+  "sha256" TEXT NOT NULL,
+  "created_at" TIMESTAMPTZ NOT NULL,
+  "updated_at" TIMESTAMPTZ NOT NULL
+);
+
+CREATE INDEX idx_connector_images_image ON connector_images("image");
+CREATE UNIQUE INDEX idx_connector_images_image_tag ON connector_images("image", "tag");
+CREATE UNIQUE INDEX idx_connector_images_sha256 ON connector_images("sha256");
```

Comment:
I'm not sure if this is a good idea. It's allowed and expected for multiple tags to all have the same sha256, and it's not clear that we need to prevent duplicate sha sums. Totally fine to leave this for now, since I gather that this is sort of just a placeholder, anyway, but it's making me think that perhaps it would be a good time to talk through how we want to model connectors in this system a little more holistically.

Reply:
That's a really good point. This was just a mistake on my part. I'll remove that index.

I'd agree that some of these things *are* placeholders for this schema, so I'm more than happy to talk through which things I think we don't have strong opinions on yet vs which things are Good Enough for the stage we're at.